src/Orchard.Web/Modules/Orchard.Taxonomies/Services/TaxonomyService.cs

```diff
@@ -185,6 +185,12 @@ public class TaxonomyService : ITaxonomyService {
             return TermPart.Sort(result);
         }
 
+        public int GetTermsCount(int taxonomyId) {

```

Comment:
We need this for autocomplete as we do not load the full list of terms bc they are not required for building the shape